Functions/Import/xASL_imp_NII2BIDS_RunAnat.m

```diff
@@ -26,7 +26,7 @@ function xASL_imp_NII2BIDS_RunAnat(imPar, bidsPar, studyPar, subjectSessionLabel
     
 
     %% 1. Define the pathnames
-    if length(listRuns)>1
+    if (length(listRuns)>1) || (str2num(listRuns{1}(5))>1)
```

Comment:
One step at a time. Right now we follow BIDS recommendation of not adding run when not necessary. We can make another issue that always adds run, but this will mean that we need to do more tests and change the entire Flavor-references. So I would not do it now in an issue that needs a quick fix for another dataset. But please feel free to make this issue.

Reply:
That is a BIDS option right, are you sure it is a recommendation? I would find that weird. We will run into multiple bugs this way. Think of the dataset where somebody imports a single visit or run, and then adds the second manually...

Reply:
Maybe not a real recommendation - but it is often missing in their examples and it is a valid solution. But I agree that this creates problem with dataset expansion. So I'm fine with changing that - just in a separate issue. See my justification of this.